index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = algoliasearch;
+
+function algoliasearch(applicationID, apiKey, opts) {
+  var AlgoliaSearch = require('./src/algoliasearch');
+
+  return new AlgoliaSearch(applicationID, apiKey, opts);
```

Comment:
I mean that this code could move to the function that does the new instead of the constructor. This is an idea and not something blocking at all.